docs/builders/plugin.mdx

```diff
@@ -0,0 +1,103 @@
+---
+description: >
+  The Nutanix packer plugin allow to use Packer on Nutanix AHV platform.
+page_title: Nutanix plugin
+nav_title: Nutanix
+---
+
+# Nutanix Builder
+
+This document is going to detail all Nutanix plugin source parameters.
+
+## Principle
+The Nutanix plugin will create a temporary VM as foundation of your Packer image, apply all providers your want to customize your image, then clone the VM disk image as your final Packer image.
+
+## Environment configuration
+These parameters allow to define information about platform and temporary VM used to create the image.
+
+### Required
+  - `nutanix_username` (string) - User used for Prism Central login 
+  - `nutanix_password` (string) - Password of this user for Prism Central login 
+  - `nutanix_endpoint` (string) - Prism Central FQDN or IP
+  - `cluster_name` (string) - Nutanix cluster name used to create and store image
+  - `os_type` (string) - OS Type ("Linux" or "Windows")
+  
+### Optional
+  - `nutanix_port` (integer) - Port configured on Prism Central to connect on
+  - `nutanix_insecure` (boolean) - Authorize connection on Prism Central without valid certificate
+  - `cpu` (integer) - Number of vCPU for temporary VM
+  - `memory_mb` (integer) - Size of vRAM for temporary VM (in megabytes)
+  - `cd_files` (array of strings) - A list of files to place onto a CD that is attached when the VM is booted. This can include either files or directories; any directories will be copied onto the CD recursively, preserving directory structure hierarchy
+  - `cd_label` (string) - Label of this CD Drive
+
+## Output configuration
+These parameters allow to configure everything around image creation, from the temporary VM connection to the final image definition.
+
+### All OS
+- `image_name` (string) - Name of the output image
+- `shutdown_command` (string) - Command line to shutdown your temporary VM
+- `shutdown_timeout` (string) - Timeout for VM shutdown (format : 2m)
+
+### Dedicated to Linux
+- `user_data` (string) - cloud-init (Linux) content base64 coded.  
+- `ssh_username` (string) - user for ssh connection initiated by Packer.
+- `ssh_password` (string) - password for the this ssh user.
+
+### Dedicated to Windows
+- `communicator` (string) - Protocol used for Packer connection (ex "winrm")
```

Comment:
communicator is not only dedicated to Windows... it is ssl/winrm and defaults to ssh if not set

Reply:
fixed in last commit

Reply:
sorry, ssl was a typo, there is only ssh/winrm
See here: https://www.packer.io/docs/communicators#communicators

Reply:
Oups. I should have seen that. Fixed in last commit.